projects/cds/src/config/cds-config-validator.spec.ts

```diff
@@ -2,6 +2,46 @@ import { CdsConfig } from './cds-config';
 import { cdsConfigValidator } from './cds-config-validator';
 
 describe('cdsConfigValidator', () => {
+  it('should warn about an undefined cds configuration section', () => {
```

Comment:
Did you make some changes here, or is the wrong diff problem happening again?

Reply:
I did make some changes here. The additional configuration for profile tag wasn't included in the configuration validator, and made the code coverage go down to around 75%, so I included some more tests

Reply:
Thanks for doing that!